CONTRIBUTING.md

```diff
@@ -0,0 +1,18 @@
+# How to contribute
+
+## Wanna report a bug or new feature?
+
+Just head over to Github issues.
+
+## Contribute to the codebase?
+
+What we like:
+
+- Small PRs.
+- Commit messages that are [clean](https://cbea.ms/git-commit/) but also [fun](https://gitmoji.dev/).
+- Writing unit tests for critical code.
+
+What we use (you should install these tools in your favorite editor):
+
+- Javascript: typescript, eslint, prettier.
+- Python: pylint, black, isort, flake8, mypy.
```

Comment:
Maybe we should mention the test suite of choice (pytest) somewhere in here as well?